pkg/bgp/speaker/speaker.go

```diff
@@ -170,6 +192,11 @@ func (s *Speaker) OnUpdateEndpoints(eps *slim_corev1.Endpoints) {
 // OnAddNode notifies the Speaker of a new node.
 func (s *Speaker) OnAddNode(node *slim_corev1.Node) error {
 	log.Infof("chris Speaker OnAddNode %v", node.GetName())
+
+	if s.shutDown() {
+		return fmt.Errorf("cannot enqueue event, speaker is shutdown")
```

Comment:
Probably worth extracting out this error into its own variable via `errors.New()`

```
var (
  errShuttingDown = errors.New(...)
)
```

Reply:
I like that :+1: